pontoon/base/static/js/main.js

```diff
@@ -244,6 +244,9 @@ $(function () {
         pathname: window.location.pathname,
       },
     );
+
+    // Initialize the timego plugin on the notifications
+    $('#notifications').timego();
```

Comment:
This line should go out of the if-block.

The selector should also be made more specific and only target the very element that will show the data:
https://timeago.yarp.com/

Then you might get a JS error, because the library still needs to be included.